cpp/src/groupby/sort/groupby.cu

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019-2020, NVIDIA CORPORATION.
+ * Copyright (c) 2019-2021, NVIDIA CORPORATION.
```

Comment:
possible merge conflict with PR #7387

Reply:
I see. Then either this, or that PR need to be merged first.